service/internal/extensions/extensions.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package builder
+package extensions
```

Comment:
I wonder if it would be better to put all builder packages under the existing builder package, e.g. `service/internal/builder/extensions`? It may be a bit too much nesting but `service/internal/extensions` is not very clear.

Reply:
It is an internal package, so I would not worry too much about the path.